docs/cli-options.rst

```diff
@@ -196,6 +196,10 @@ Instaloader to login.
 
    Login name (profile name) for your Instagram account.
 
+.. option:: --load-cookies BROWSER-NAME, -lc BROWSER-NAME
```

Comment:
I would not use `-lc` as a option name, we always use single letter option names.

It would also be nice if the profile or the path to the cookie file could be set by the user, similar to the provisional cookie import script mentioned in our documentation, as there are many users who have multiple profiles in use.

Reply:
Nice, I think `-b` and `-B` are fine.